ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -102,6 +102,18 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     Ok(views.html.recentactivity(userEventDBs))
   }
 
+  def statusDistribution = AuthAction { implicit request =>
+    val distribution: Map[RecipeStatus, Long] = Map(
+      New -> db.countRecipesInGivenStatus(New),
```

Comment:
Again, it's worth being mindful that you are making 6 calls to the DB here when this (if we cared about performance) could probably be achieved with a single query partitioned server side.